changelogs/current.yaml

```diff
@@ -18,6 +18,16 @@ behavior_changes:
   change: |
     Fixed metric tag extraction so that worker_id is properly extracted from the listener stats. This changes the Prometheus name from
     envoy_listener_worker_1_downstream_cx_active{envoy_listener_address="0.0.0.0_10000"} to envoy_listener_downstream_cx_active{envoy_listener_address="0.0.0.0_10000", envoy_worker_id="1"} .
+- area: stats http local_rate_limit
+  change: |
+    Fixed metric tag extraction so that :ref:`stat_prefix <envoy_v3_api_field_extensions.filters.http.local_ratelimit.v3.LocalRateLimit.stat_prefix>`
+    is properly extracted. This changes the Prometheus name from
+    envoy_myprefix_http_local_rate_limit_rate_limited{} to envoy_http_local_rate_limit_rate_limited{envoy_local_http_ratelimit_prefix="myprefix"}.
```

Comment:
I think the old-name here is incorrect. Shouldn't it be `envoy_http_local_rate_myprefix_limit_rate_limited{}`